### PR TITLE
Javascript/javascript3/week3

### PR DESCRIPTION
**Pull request by soujanyapolavarapu**

### CODE REVIEW

File: javascript/javascript3/week3/Getting-into-promises.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
+/*
+const li1 = document.createElement('li');
+ li1.innertext=promise.items;
+li1.appendChild(ul);*/
+const url1 = 'https://api.github.com/search/repositories?q=user:rajanibusani';
+const url2 = 'https://api.github.com/search/repositories?q=user:JyothiNandyala';
+const url3 = 'https://api.github.com/search/repositories?q=user:sofiiadidovych';
```
Comment (aleksd-dfds) [inline javascript/javascript3/week3/Getting-into-promises.js:7]: similar URL is used 3 times. Can extract the long part of the URL into its own const

File: javascript/javascript3/week3/Shopping-cart-using-classes.js [hunk 2]
```diff
@@ -0,0 +1,122 @@
+class Product {
+  constructor(name, price) {
+    this.name = name;
+    this.price = price;
+  }
+
+  convertToCurrency(currency){
+  const url='https://api.exchangeratesapi.io/latest?base=DKK';
+   fetch('https://api.exchangeratesapi.io/latest?base=DKK')
```
Comment (aleksd-dfds) [inline javascript/javascript3/week3/Shopping-cart-using-classes.js:9]: why not use the const from above?

File: javascript/javascript3/week3/Shopping-cart-using-classes.js [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+  });
+  //fot total price of products
+  const total = document.createElement('h3');
+  total.textContent  = 'Total Price of Items : ' + this.getTotal();
+  cart.append(total);
+  }
+
+  getUser(){
+    const api= 'https://jsonplaceholder.typicode.com/users/1';
+    fetch(api)
```
Comment (aleksd-dfds) [inline javascript/javascript3/week3/Shopping-cart-using-classes.js:65]: yes

**Review by aleksd-dfds — APPROVED** [review]

File: javascript/javascript3/week3/Getting-into-promises.js (hunk 1 above)
Comment (soujanyapolavarapu) [reply javascript/javascript3/week3/Getting-into-promises.js:7]: Thankyou for your comments aleks.I will try to modify s soon as possible.

**Pull request merged**